### ISSUE TITLE
No notification in the activity centre for everyone mention

### ISSUE THREAD

**Issue by churik** [status-im/status-react#15147]
# Bug Report

## Problem 
In https://github.com/status-im/status-mobile/pull/15094 ``@everyone`` mention was added, but there is still no notification in the activity center about it.
However, the notification was shown on rc-1 build, but no on `develop` on 21/02/22 

#### Expected behavior
can see a notificaton in the activity center (added screenshot from rc-1)
<img width="228" alt="photo_2023-02-21 13 08 24 jpeg 2023-02-21 13-08-55" src="https://user-images.githubusercontent.com/4557972/220341327-8077882a-ec76-4d8d-b929-62f9f69fa7a7.png">

#### Actual behavior
no notification

### Reproduction
**Prerequisites:** mobile and desktop clients are joined the same community
1) From desktop: send ``@everyone`` mention
2) From mobile: check home screen

### Additional Information

- Status version: nightly 21/02/23
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

**Comment by churik** [comment]
closed as duplicate of https://github.com/status-im/status-mobile/issues/15615
